book/docs/Open_data_code.ipynb

```diff
@@ -2,21 +2,77 @@
  "cells": [
```

Comment:
<p>Should we mention something like Zenodo for being able to get a DOI for a github repo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/scaling_science/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done!</p><p> </p>